src/towncrier/test/test_check.py

```diff
@@ -204,6 +213,44 @@ def test_fragment_exists_but_not_in_check(self):
                 (result.output, str(fragment_path)),
             )
 
+    def test_fragment_exists_and_staged(self):
+        """A fragment that exists but is marked as check=False is ignored by the check."""
+        runner = CliRunner()
+
+        with runner.isolated_filesystem():
+            create_project(
+                "pyproject.toml",
+                main_branch="master",
+                extra_config="[[tool.towncrier.type]]\n"
+                'directory = "feature"\n'
+                'name = "Features"\n'
+                "showcontent = true\n"
+                "[[tool.towncrier.type]]\n"
+                'directory = "sut"\n'
+                'name = "System Under Test"\n'
+                "showcontent = true\n"
+                "check=false\n",
```

Comment:
I think you could shorten this test by omitting this `check=False` stuff (and staging the `.sut` file below).

Reply:
fixed in 363532aa92fc31c5d46c6bea2cd4760a7604c4f9